### PR TITLE
WEBUI-835: add translations for collection events

### PR DESCRIPTION
**Pull request by tmartins**

### CODE REVIEW

File: i18n/messages.json [hunk 1]
```diff
@@ -572,6 +572,8 @@
   "eventType.workflowTaskCompleted": "Workflow task completed",
   "eventType.workflowTaskEnded": "Workflow task ended",
   "eventType.workflowTaskRejected": "Workflow task rejected",
+  "eventType.addedToCollection":"Document added to collection",
+  "eventType.removedFromCollection":"Document removed from collection",
```
Comment (Gabez0r) [inline i18n/messages.json]: The JSON keys should be sorted alphabetically.

**Review by Gabez0r — COMMENTED** [review]
Thank you for the help! 👍

File: i18n/messages.json [hunk 2]
```diff
@@ -528,6 +528,7 @@
   "eventCategory.eventWorkflowCategory": "Approval workflow",
   "eventCategory.NuxeoAuthentication": "Authentication",
   "eventCategory.relationNotificationCategory": "Waiting publication",
+  "eventType.addedToCollection":"Document added to collection",
```
Comment (poonamyadav252) [inline i18n/messages.json]: It looks nice, but it would be better if we can provide space after ":" as other keys have.

File: i18n/messages.json [hunk 3]
```diff
@@ -561,6 +562,7 @@
   "eventType.loginFailed": "Login failed",
   "eventType.loginSuccess": "Login successful",
   "eventType.logout": "Logout",
+  "eventType.removedFromCollection":"Document removed from collection",
```
Comment (poonamyadav252) [inline i18n/messages.json]: Same as above :)

**Review by swarnadipa-dev — APPROVED** [review]
Looks good

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**